README.md

```diff
@@ -5,6 +5,19 @@
 
 Go client and cli for [outline](https://www.getoutline.com/).
 
+# Installation
+
+## Installing the client:
+```shell
+$ go get github.com/ioki-mobility/go-outline
```

Comment:
Here you added the dollar sign, but not below. Is that intended? Otherwise I would suggest to remove it her 🙃

```suggestion
go get github.com/ioki-mobility/go-outline
```

Reply:
ahh stupid copy paste typo